backend/models.py

```diff
@@ -3,7 +3,6 @@
 class Student(db.Model):
 	__tablename__ = 'student'
 	id = db.Column(db.Integer(), primary_key=True)
-	studentId = db.Column(db.Integer())
```

Comment:
When I request '/students' sever returns:
AttributeError: 'Student' object has no attribute 'studentId'

Reply:
Otherwise rest of the routes look gud

Reply:
Fixed in latest push.